pyproject.toml

```diff
@@ -7,8 +7,8 @@ requires = [
     "cython>=0.29.20,<3.0.0; python_version<='3.9'",
     # See https://numpy.org/doc/stable/user/depending_on_numpy.html for
     # the recommended way to build against numpy's C API:
-    "oldest-supported-numpy",
-    "scipy>=1.8",
+    "numpy>=2.0.0",
```

Comment:
Is building with the latest numpy2 the correct thing to do here? Or should we build with an old numpy2 like with oldest-supported-numpy for numpy 1.x?

Reply:
This is how I understand their migration guide. There is no `oldest-supported-numpy2` and that package has not been updated for the release.

Reply:
Thanks @rgommers !